### PR TITLE
add SomaticSeq

### PR DESCRIPTION
**Pull request by DavyCats**

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -14,3 +14,4 @@ dependencies:
   - biopet-scatterregions==0.1
   - tabix==0.2.5
   - vardict-java==1.5.4
+#  - somaticseq==2.8.1
```
Comment (rhpvorderman) [inline environment.yml]: Yes. With containers, where do we record the used versions? That is the thing that I am going to miss with environment.yml

File: environment_strelka.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+name: strelka
```
Comment (rhpvorderman) [inline environment_strelka.yml]: How do you plan to go about activating two environments at once?

File: somatic-variantcalling.wdl [hunk 3]
```diff
@@ -48,13 +56,114 @@ workflow SomaticVariantcalling {
...
+                normalBam = select_first([controlBam]),
+                mutect2VCF = mutect2.outputVCF.file,
+                vardictVCF = vardict.outputVCF.file,
+                strelkaSNV = strelka.variantsVCF.file,
+                strelkaIndel = select_first([strelka.indelsVCF]).file
+        }
+    }
+
+    if (defined(trainingSet) && !defined(controlBam)) {
+        TrainingSet trainSetSingle = select_first([trainingSet]) #FIXME workaround 'no such field'
```
Comment (rhpvorderman) [inline somatic-variantcalling.wdl]: Could you elaborate? which error is being worked around here?

**Review by rhpvorderman — CHANGES_REQUESTED** [review]
Looks good. Just a few small things.

File: environment_strelka.yml (hunk 2 above)
Comment (DavyCats) [reply environment_strelka.yml]: We don't, this was a workaround thing from before singularity.

File: environment.yml (hunk 1 above)
Comment (DavyCats) [reply environment.yml]: We could keep the environment.yml as a list of versions we used to test.

**Review by rhpvorderman — APPROVED** [review]

**Pull request merged**